src/main/java/me/contaria/seedqueue/mixin/client/MinecraftClientMixin.java

```diff
@@ -542,7 +543,7 @@ private boolean fastQuit(IntegratedServer server, Operation<Boolean> original) {
     )
     private void logSystemInformation(CallbackInfo ci) {
         if (Boolean.parseBoolean(System.getProperty("seedqueue.logSystemInfo", "true"))) {
```

Comment:
Could you move the system property check into `SeedQueueSystemInfo#logSystemInformation` to keep it consistent with config logging?

Reply:
made all the changes, thanks for reviewing :D